Userland/Libraries/LibWeb/OutOfProcessWebView.cpp

```diff
@@ -163,6 +163,7 @@ void OutOfProcessWebView::handle_resize()
 void OutOfProcessWebView::keydown_event(GUI::KeyEvent& event)
 {
     client().async_key_down(event.key(), event.modifiers(), event.code_point());
+    event.ignore();
```

Comment:
Oh frick this is a really tricky one.. What does this mean for out-of-process browser engines..
We will now send all key events to the web content, even stuff that's potentially UI actions, like Ctrl+T ("open new tab") will become a JavaScript event despite the user *most likely* wanting to open a new tab..